packages/teams-js/src/private/videoEx.ts

```diff
@@ -244,6 +249,20 @@ export namespace videoEx {
     }
   }
 
+  function createMonitoredVideoFrameHandler(
+    videoFrameHandler: VideoFrameHandler,
+    videoPerformanceMonitor?: VideoPerformanceMonitor,
```

Comment:
where are you passing videoPerformanceMonitor into this function? It doesn't seem like you have anywhere that this is actually being specified

Reply:
I wonder if this function is getting confused and using the one defined as a const up at the top of the file? Either way, something confusing going on.